package.json

```diff
@@ -18,53 +18,53 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@babel/core": "7.10.5",
+    "@babel/core": "7.11.6",
     "@babel/plugin-proposal-class-properties": "7.10.4",
-    "@babel/plugin-proposal-object-rest-spread": "7.10.4",
+    "@babel/plugin-proposal-object-rest-spread": "7.11.0",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
-    "@babel/preset-env": "7.10.4",
+    "@babel/preset-env": "7.11.5",
     "@kazupon/vue-i18n-loader": "0.5.0",
     "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "10.0.3",
+    "babel-eslint": "10.1.0",
     "babel-jest": "24.9.0",
     "babel-loader": "8.1.0",
     "babel-merge": "3.0.0",
     "babel-plugin-dynamic-import-node": "2.3.3",
     "babel-plugin-lodash": "3.3.4",
     "css-loader": "3.4.2",
     "eslint": "6.8.0",
-    "eslint-config-airbnb-base": "14.0.0",
-    "eslint-plugin-import": "2.20.0",
+    "eslint-config-airbnb-base": "14.2.0",
+    "eslint-plugin-import": "2.22.1",
     "eslint-plugin-vue": "6.1.2",
     "fstream": "1.0.12",
     "jest": "24.9.0",
     "jest-serializer-vue": "2.0.2",
     "lodash-es": "4.17.15",
-    "lru-cache": "5.1.1",
+    "lru-cache": "6.0.0",
```

Comment:
I think we might want to only do this one in this PR.  If we do the others (webpack, vue, eslint, I think we'll potentially get the slipstream deps out of sync with these deps).

Since w'ell need to do multiple rouds, and we don't want multiple breaking changes - maybe the way we tackle this is to do them with small beta tags?  Right now we're at 4.8.1, so maybe we do like

* 5.0.0-beta.1 - express deps along side a slipstream sibling
* 5.0.0-beta.2 - vue deps along side a slipstream sibling
* and so on

And then once we're through them all we can release a v5?

Reply:
Also don't forget to update the `lru-cache` `peerDependency` to `^6.0.0` below to stay synced with this

Reply:
I reverted my changes and made the update to lru-cache and its peer dependency and bumped the version to 5.0.0-beta.1.